sql/moz-fx-data-shared-prod/telemetry_derived/newtab_interactions_v1/query.sql

```diff
@@ -0,0 +1,130 @@
+WITH events_unnested AS (
+  SELECT
+    DATE_TRUNC(submission_timestamp, DAY) AS submission_date,
+    category AS event_category,
+    name AS event_name,
+    timestamp AS event_timestamp,
+    client_info,
+    metadata,
+    normalized_os,
+    normalized_os_version,
+    normalized_country_code,
+    normalized_channel,
+    ping_info,
+    extra AS event_details,
+    metrics
+  FROM
+    `moz-fx-data-shared-prod.firefox_desktop_stable.newtab_v1`,
+    UNNEST(events)
+  WHERE
+    DATE(submission_timestamp) = @submission_date
+    AND category IN ('newtab', 'topsites', 'newtab.search', 'newtab.search.ad')
```

Comment:
for safety should we be doing `LOWER(category)`?
same for name below?

Reply:
These are usually standardized/version-controlled in the client I think this should be fine. Though I will add a link to the Glean dictionary entry